textdb/textdb-common/src/main/java/edu/uci/ics/textdb/common/field/ListField.java

```diff
@@ -38,7 +38,7 @@ public boolean equals(Object obj) {
         if (list == null) {
             if (other.list != null)
                 return false;
-        } else if (!list.equals(other.list))
+        } else if (!list.containsAll(other.list))
```

Comment:
Because even though the list has same number of elements, with same size, equals return false, where as containsAll returns True

other.list.containsAll(list) Returns True;
list.containsAll(others.list) Returns True;
other.list.equals(list) Returns False